src/main/java/seedu/duke/data/notebook/Note.java

```diff
@@ -20,11 +20,10 @@
      * @param content of the note.
      * @param isPinned status of the note.
      */
-    public Note(String title, ArrayList<String> content, Boolean isPinned, boolean isArchived) {
+    public Note(String title, ArrayList<String> content, Boolean isPinned) {
         this.title = title;
         this.content = content;
         this.isPinned = isPinned;
-        this.isArchived = isArchived;
```

Comment:
Is the boolean added back in? Looks like it's still removed.

Reply:
Updated, request for a final review prior to merging